client/src/locales/en/translation.json

```diff
@@ -129,7 +129,7 @@
   "com_auth_username_max_length": "Username must be less than 20 characters",
   "com_auth_username_min_length": "Username must be at least 2 characters",
   "com_auth_verify_your_identity": "Verify Your Identity",
-  "com_auth_welcome_back": "Welcome back",
+  "com_auth_welcome_back": "Welcome to Edge",
```

Comment:
I suggest adding a new key instead of modifying the key. Then create an environment variable to point to the new key. If the env var is set, use it, otherwise use the default key.

This way if we use this for another client (or for our own instance), it won't say Edge. Also, it would allow us to localize the branding if that was ever desired.